### ISSUE TITLE
How to get directory listing ?

### ISSUE THREAD

**Issue by mayuresh82** [codeskyblue/gohttpserver#144]
I am trying to use a gohttpserver as a remote source for rclone. [https://rclone.org/http/] requires directory listing returned in index.html.  Is there a way to get remote directory listing ?

**Comment by codeskyblue** [comment]
add &json=true to path suffix

https://localhost:8000/folder/?json=true

**Comment by mayuresh82** [comment]
@codeskyblue  i dont think that is going to work with rclone which requires the directory listing inside index.html :(

**Comment by 457452950** [comment]
Your description is very unclear. If possible, a screenshot of the scene would be very helpful.
